apps/web/modules/auth/signup/lib/team.ts

```diff
@@ -63,3 +70,73 @@ export const createTeamMembership = async (invite: CreateMembershipInvite, userI
     throw error;
   }
 };
+
+const getDefaultTeam = reactCache(
```

Comment:
please make a generic service here `getTeam` that takes the `teamId` and `organizationId` and fetches the required data with cache 🙏